docs/reference/search/multi-search.asciidoc

```diff
@@ -78,7 +78,8 @@ include::{docdir}/rest-api/common-parms.asciidoc[tag=index-ignore-unavailable]
 
 `max_concurrent_searches`::
 (Optional, integer)
-Maximum number of concurrent searches the multi search API can execute.
+Maximum number of concurrent searches the multi search API can execute. Defaults
+to +max(1, (# of <<data-node,data nodes>> * min(# of <<node.processors,allocated processors>>, 10)))+.
```

Comment:
This part: `<<node.processors,allocated processors>>` is more like: `((<<node.processors,allocated processors>> * 3) / 2) + 1`. But perhaps we just refer to the search thread default size?

Reply:
Good catch. I've replaced `<<node.processors,allocated processors>>` with `<<search-threadpool,search thread pool size>>` and added an appropriate anchor to the search thread pool definition.

Thanks @martijnvg!